### PR TITLE
Show contacts on Card hover

### PR DESCRIPTION
**Pull request by filipebfernandes**
Add contacts to Association Card on hover, and `visit page` link to the corresponding association page.

### CODE REVIEW

File: src/components/Association/index.css [hunk 1]
```diff
@@ -1,7 +1,82 @@
...
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 10px;
+
+  background: rgba(40, 40, 59, 0.8);
```
Comment (jfranciscosousa) [inline src/components/Association/index.css]: Can you assign colors to a variable in this file? So we know what color it is just by looking at the name.

Comment (filipebfernandes) [reply src/components/Association/index.css]: yes

File: src/components/Association/index.js [hunk 2]
```diff
@@ -16,30 +27,100 @@ class Association extends Component {
...
+      site: PropTypes.string,
+      phones: PropTypes.arrayOf(PropTypes.string),
+      email: PropTypes.string,
+      image: PropTypes.object,
+      address: PropTypes.string,
     }).isRequired,
   };
 
-  render() {
+  renderContactsOnHover = () => {
```
Comment (jfranciscosousa) [inline src/components/Association/index.js]: You can extract this to a local component like `Association/Contacts.js`

Comment (jfranciscosousa) [reply src/components/Association/index.js]: Otherwise the component will end up very big

Comment (filipebfernandes) [reply src/components/Association/index.js]: Ok

File: src/components/Association/Contacts.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
+$background-color: rgba(40, 40, 59, 0.8);
```
[the remark this answers is not in the document]
Comment (filipebfernandes) [reply src/components/Association/Contacts.css]: 👍

**Pull request merged**